### PR TITLE
Secure Migration Adding New Office Users #162792520

### PR DESCRIPTION
**Pull request by donaldthai**
## Description

This adds new office users per `https://www.pivotaltracker.com/story/show/162792520`.

## Setup

To run these migrations on your database and verify the data, run:

bin/run-prod-migrations, then inspect the prod_migrations database that was created in your local postgres.
You should see office_users for every person's last name, first name, and email from the pivotal story.

## Code Review Verification Steps

* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](./docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/162792520) for this change

### CODE REVIEW

File: local_migrations/20181220182247_add-office-users-#162792520.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+
+INSERT INTO public.office_users VALUES (uuid_generate_v4(), NULL, 'McTest', concat('Test-', uuid_generate_v4()), NULL, concat('Test-', uuid_generate_v4(), '@example.com'), '', '0931a9dc-c1fd-444a-b138-6e1986b1714c', now(), now());
+INSERT INTO public.office_users VALUES (uuid_generate_v4(), NULL, 'McTest', concat('Test-', uuid_generate_v4()), NULL, concat('Test-', uuid_generate_v4(), '@example.com'), '', '0931a9dc-c1fd-444a-b138-6e1986b1714c', now(), now());
+INSERT INTO public.office_users VALUES (uuid_generate_v4(), NULL, 'McTest', concat('Test-', uuid_generate_v4()), NULL, concat('Test-', uuid_generate_v4(), '@example.com'), '', '0931a9dc-c1fd-444a-b138-6e1986b1714c', now(), now());
+INSERT INTO public.office_users VALUES (uuid_generate_v4(), NULL, 'McTest', concat('Test-', uuid_generate_v4()), NULL, concat('Test-', uuid_generate_v4(), '@example.com'), '', '0931a9dc-c1fd-444a-b138-6e1986b1714c', now(), now());
+INSERT INTO public.office_users VALUES (uuid_generate_v4(), NULL, 'McTest', concat('Test-', uuid_generate_v4()), NULL, concat('Test-', uuid_generate_v4(), '@example.com'), '', '0931a9dc-c1fd-444a-b138-6e1986b1714c', now(), now());
```
Comment (chrisgilmerproj) [inline local_migrations/20181220182247_add-office-users-#162792520.sql]: I'm not sure you need this migration to do anything.  We just need an empty file that has the same name and maybe runs `SELECT(1)`.  Can you connect with @kimallen and see what her last one of these did?

Comment (donaldthai) [reply local_migrations/20181220182247_add-office-users-#162792520.sql]: @chrisgilmerproj just updated with an empty local migration file

**Comment by chrisgilmerproj** [comment]
You should be able to test this by running `./bin/run-prod-migrations`.  Did you give that a shot? Did the users show up?

**Comment by donaldthai** [comment]
Yup, users are showing up for me!

**Pull request merged**